aiida/cmdline/commands/graph.py

```diff
@@ -34,65 +31,57 @@ def __init__(self):
         A dictionary with valid subcommands as keys and corresponding functions as values
         """
 
-        # Usual boilerplate to set the environment
-        if not is_dbenv_loaded():
-            load_dbenv()
-
         self.valid_subcommands = {
-            'generate': (self.graph_generate, self.complete_none)
+            'generate': (self.cli, self.complete_none)
             # TODO: add a command to find connections between two points
```

Comment:
This should probably be turned into a feature request issue instead of a TODO (Doesn't the linter complain about TODO comments?)

Reply:
Agreed. Removed and I will open an issue.